client/dashboard/me/notifications-extras/test/index.test.tsx

```diff
@@ -0,0 +1,370 @@
+/**
+ * @jest-environment jsdom
+ */
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import nock from 'nock';
+import Snackbars from '../../../app/snackbars';
+import { render as dashboardRender } from '../../../test-utils';
+import NotificationsExtras from '../index';
+import type { UserNotificationSettings, WpcomNotificationSettings } from '@automattic/api-core';
+
+const defaultWpcomSettings: WpcomNotificationSettings = {
+	marketing: false,
+	research: false,
+	affiliates: false,
+	community: false,
+	promotion: false,
+	news: false,
+	digest: false,
+	reports: false,
+	news_developer: false,
+	wpcom_spain: false,
+	scheduled_updates: false,
+	learn: false,
+	a4a_agencies: false,
+	jetpack_agencies: false,
+	jetpack_manage_onboarding: false,
+	jetpack_marketing: false,
+	jetpack_research: false,
+	jetpack_promotion: false,
+	jetpack_news: false,
+	jetpack_reports: false,
+	akismet_marketing: false,
+	woopay_marketing: false,
+	gravatar_onboarding: false,
+};
+
+const defaultUserSettings: UserNotificationSettings = {
+	blogs: [],
+	other: {
+		timeline: { comment_like: false, comment_reply: false },
+		email: { comment_like: false, comment_reply: false },
+		devices: [],
+	},
+	wpcom: defaultWpcomSettings,
+};
+
+const mockGetNotificationSettingsApi = (
+	settings: UserNotificationSettings = defaultUserSettings
+) => {
+	return nock( 'https://public-api.wordpress.com:443' )
+		.get( '/rest/v1.1/me/notifications/settings' )
+		.reply( 200, settings );
+};
+
+const mockSaveNotificationSettingsApi = (
+	expectedSettings: Partial< WpcomNotificationSettings >
+) => {
+	return nock( 'https://public-api.wordpress.com:443' )
+		.post( '/rest/v1.1/me/notifications/settings', {
+			wpcom: expectedSettings,
+		} )
+		.reply( 200, { wpcom: expectedSettings } );
+};
+
+const notificationSnackBar = () => {
+	// Snackbar requires a custom matcher because its aria-live is not supported by the testing library
+	return document.getElementById( 'a11y-speak-polite' );
+};
+
+describe( 'NotificationsExtras', () => {
+	beforeEach( () => {
+		nock.disableNetConnect();
+		nock.cleanAll();
+		// Snackbar requires window.scrollTo to be defined
+		window.scrollTo = jest.fn();
+	} );
+
+	afterEach( () => {
+		nock.cleanAll();
+	} );
+
+	it( 'renders the page header and sections correctly', async () => {
```

Comment:
Considering there is no logic managing whether the `Extras` title is rendered or not, we don't need this test.